### ISSUE TITLE
Has incubation stalled?

### ISSUE THREAD

**Issue by marcoscaceres** [ulan/performance-measure-memory#29]
Hi, as there appears to have been little activity in this repo in the last few years, we wanted to check if this incubation is still active. 

If the incubation is no longer active, we would like to consider archiving the repo in a few weeks. 

Let us know!

**Comment by domenic** [comment]
I don't think it's a good idea to archive repositories for specifications that are shipping and used in the wild, even if there have been no bugs reported with them or no second-vendor interest.

**Comment by marcoscaceres** [comment]
Yeah, that's fair. I didn't know this had shipped (or that is had shipped unpref'ed on the web without a second implementation 😒). 

We should figure the above out in the future though. Personally, I'm not sure about the WICG should be a forever home for single vendor things - but don't have a good solution for what we should do when we get into this situation.

**Comment by yoavweiss** [comment]
This has indeed shipped in Chromium, and we should [label](https://github.com/WICG/admin/issues/138) it as such.
There was [interest from Mozilla at TPAC 2021](https://w3c.github.io/web-performance/meetings/2021/2021-10-TPAC/index.html#h.glex7ka68pwu), but it was considered "too early to adopt".

^^ @bas-moz @bdekoz @sefeng211

**Comment by yoavweiss** [comment]
> We should figure the above out in the future though. Personally, I'm not sure about the WICG should be a forever home for single vendor things - but don't have a good solution for what we should do when we get into this situation.

Let's move that to https://github.com/WICG/admin/issues/138
